pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGenerator/indexScript.js

```diff
@@ -0,0 +1,34 @@
+Behaviour.specify(".prototype-button-reference-holder", 'prototype', 0, function (e) {
```

Comment:
```suggestion
Behaviour.specify(".directive-generator-button-reference-holder", 'prototype', 0, function (e) {
```

Reply:
Needed to change the class in jelly file as well, done